### ISSUE TITLE
[Bug]: (a11y)-Multiple accessibility issues with React-Router

### ISSUE THREAD

**Issue by akash07k** [remix-run/react-router#11915]
### What version of React Router are you using?

6.26.1

### Steps to Reproduce

1. Implement React-router in any app.
2. Define couple of the routes
3. Run the app
4. Click on any of the link and observe the screen reader announcement and focus

### Expected Behavior

Screen reader should announce tht title of the newly loaded page

### Actual Behavior

Whenever we click on any route (`<link>`), the page title of the newly loaded page is not announced.
Also, the focus is set to somewhere randomly on the page.

**Comment by timdorr** [comment]
The page title is up to you, as there are different ways of defining that and triggering the update. Focus is also dependant on how you've set up your page and isn't something the library can infer as easily. I would recommend a layout route to centralize these things: https://reactrouter.com/en/main/start/concepts#layout-routes

**Comment by akash07k** [comment]
Pretty unprofessional and weird response.
Triggering the update announcement should be handled at library side itself as well as the focus management.
But no problem, seems I'm better using another better library if I anyways need to do all the heavy lifting myself.
Using a library from unprofessional authors is anyways not good and is a waste of time.

> The page title is up to you, as there are different ways of defining that and triggering the update. Focus is also dependant on how you've set up your page and isn't something the library can infer as easily. I would recommend a layout route to centralize these things: https://reactrouter.com/en/main/start/concepts#layout-routes
